### PR TITLE
Actions queue

### PR DESCRIPTION
**Pull request by ssestak**
Add `ActionsQueue` into `FuturedHelpers`.

Update `ModalCoverModel` to able presenting modal screens via `ActionsQueue`.

### CODE REVIEW

File: Sources/FuturedArchitecture/Architecture/QueueAction.swift [hunk 1]
```diff
@@ -0,0 +1,39 @@
+//
+//  File.swift
```
Comment (samoilyk) [inline Sources/FuturedArchitecture/Architecture/QueueAction.swift]:
```suggestion
//  QueueAction.swift
```

File: Sources/FuturedHelpers/ActionsQueue/ActionsCache.swift [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+//  Created by Simon Sestak on 11/10/2024.
+//
+
+import FuturedArchitecture
+import Combine
+
+actor ActionsCache<A: QueueAction> {
+    let subject: CurrentValueSubject<A?, Never> = CurrentValueSubject(nil)
+
+    private var actions = [A]() {
```
Comment (samoilyk) [inline Sources/FuturedHelpers/ActionsQueue/ActionsCache.swift:14]: jinak mozna se da zbavit toho subjectu tim, ze se z toho udela @Published property. Da se na to subscribovat, drzi to current value a bude to o par radku mensi

Comment (ssestak) [reply Sources/FuturedHelpers/ActionsQueue/ActionsCache.swift:14]: Skúsil som to ale potom by musel byť  `ActionsQueue.init` asynchrónny nie ? 
A to kvôli týmto pár riadkom (viz screenshot). 

image

Comment (samoilyk) [reply Sources/FuturedHelpers/ActionsQueue/ActionsCache.swift:14]: mrkneme se na to spolu dneska 👍

**Pull request closed without merging**